DigitalOcean Docs / Rename DigitalOcean to dobs

akutz opened the pull request:
This patch adds documentation for the DigitalOcean Block Storage driver as well as renaming said driver to `dobs`.

codenrhoden commented on .docs/user-guide/storage-providers.md:
````diff
@@ -656,14 +656,53 @@ libstorage:
+```yaml
+dobs:
+  token: 123456
+  region: us-east-1
````
I'd prefer to have this be set to a real region that supports Volumes. Either `nyc1` or `sfo2`.

codenrhoden commented on .docs/user-guide/storage-providers.md:
```diff
@@ -656,14 +656,53 @@ libstorage:
+#### Requirements
+The DigitalOcean block storage driver has the following requirements:
+
+* Valid DigitalOcean account
```
I'd add that you need an access token, and here is the link/doc on how to generate one: https://www.digitalocean.com/community/tutorials/how-to-use-the-digitalocean-api-v2#how-to-generate-a-personal-access-token

akutz replied:
Handled! Thanks!

codenrhoden reviewed (commented):
Couple minor requests. Looks good!

akutz committed:
DigitalOcean Docs / Rename DigitalOcean to dobs

This patch adds documentation for the DigitalOcean Block Storage driver
as well as renaming said driver to `dobs`.
